ts/Core/Axis/Axis.ts

```diff
@@ -1938,6 +1938,9 @@ class Axis {
          * of the plot area. When the axis' `max` option is set or a max extreme
          * is set using `axis.setExtremes()`, the maxPadding will be ignored.
          *
+         * @productdesc {highstock}
+         * When the axis `ordinal: true`, the `maxPadding` is ignored.
```

Comment:
I think we could improve this sentence. What about the same what you used in `xAxis.ordinal`?
For example:
```
* For the ordinal axis, the `minPadding` and `maxPadding` are ignored -
* use the `overscroll` instead.
```